roles/tomcat/vars/Debian-stretch.yml

```diff
@@ -3,6 +3,7 @@
 ---
 
 tomcat_default_version: "8"
+tomcat_default_minor_version: "8.5"
 tomcat_default_service: "tomcat{{ tomcat_version }}"
 tomcat_default_conf_path: "/etc/tomcat{{ tomcat_version }}"
```

Comment:
Remove `tomcat_default_minor_version` and modify the other options as follows:

tomcat_default_version: "8.5"
tomcat_default_service: "tomcat8"
tomcat_default_conf_path: "/etc/tomcat8"